build/dependencies.props

```diff
@@ -4,6 +4,7 @@
   </PropertyGroup>
   <PropertyGroup Label="Package Versions">
     <InternalAspNetCoreSdkPackageVersion>2.1.0-preview1-15618</InternalAspNetCoreSdkPackageVersion>
+    <InternalAspNetCoreSiteExtensionSdkPackageVersion>2.1.0-preview1-15623</InternalAspNetCoreSiteExtensionSdkPackageVersion>
```

Comment:
@natemcmaster I assume this is fine but wanted to check.

Reply:
Yes, this is fine.